config/components/certificates/cert-manager/self-signed-certificates.yaml

```
@@ -0,0 +1,23 @@
+# Create a self-signed issuer for development
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: selfsigned-issuer
+  namespace: system
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: serving-cert
```

I think it makes sense to name it webhook server specific, like
```suggestion
  name: devworkspace-webhooserver-certs
```

---

Hah -- I would've assumed the kustomize name-prefixing functionality applied to all resources it creates. I'll fix.

---

Turns out this issue was due to `config/components/certificates/cert-manager` being used as a base in `/config/cert-manager` and not in `config/base`, where the name prefix is specified. I've moved namePrefix to `config/cert-manager` and `config/service-sa` to compensate.